### PR TITLE
Update validator error messaging

### PR DESCRIPTION
**Pull request by trav3711**
Make error messaging for instance and ontology validators more human-readable.

### CODE REVIEW

File: tools/validators/ontology_validator/yamlformat/validator/state_lib.py [hunk 1]
```diff
@@ -144,23 +145,23 @@ class State(findings_lib.Findings):
...
+    file_context: optional object with the config file location of this state.
   Attributes:
     name: the full name (without namespace) of this state
     description: explanation of what this state represents
-    context: the config file context for where this state was defined
+    file_context: the config file context for where this state was defined
   """
 
-  def __init__(self, name, description=None, context=None):
+  def __init__(self, name, description=None, file_context=None):
```
[the remark this answers is not in the document]
Comment (charbull) [reply tools/validators/ontology_validator/yamlformat/validator/state_lib.py:156]: I dont see docstring for this init?

File: tools/validators/ontology_validator/yamlformat/validator/subfield_lib.py [hunk 2]
```diff
@@ -248,28 +247,27 @@ class Subfield(findings_lib.Findings):
...
-    context: the config file context for where this subfield was defined.
+    file_context: the config file context for where this subfield was defined.
       Optional object with the config file location of this subfield.
 
   Returns:
     An instance of the Subfield class.
   """
 
-  def __init__(self, name, category, description=None, context=None):
+  def __init__(self, name, category, description=None, file_context=None):
```
Comment (charbull) [inline tools/validators/ontology_validator/yamlformat/validator/subfield_lib.py:257]: args?

File: tools/validators/ontology_validator/yamlformat/validator/unit_lib.py [hunk 3]
```diff
@@ -220,28 +222,28 @@ class Unit(findings_lib.Findings):
...
-  def __init__(self, name, measurement_type, is_standard=False, context=None):
+  def __init__(self, name, measurement_type, is_standard=False, file_context=None):
     """Init.
 
     Args:
       name: required string name for the unit
       measurement_type: required string indicating the unit measurement type
       is_standard: whether this is the standard unit for the measurement type
-      context: optional object with the config file location of this unit.
+      file_context: optional object with the config file location of this unit.
```
Comment (charbull) [inline tools/validators/ontology_validator/yamlformat/validator/unit_lib.py:235]: for the init you changed please add docstring as in here

**Review by charbull — CHANGES_REQUESTED** [review]
minor linter stuff

File: tools/validators/ontology_validator/yamlformat/validator/subfield_lib.py (hunk 2 above)
Comment (trav3711) [reply tools/validators/ontology_validator/yamlformat/validator/subfield_lib.py:257]: Done.

File: tools/validators/ontology_validator/yamlformat/tests/field_lib_test.py [hunk 4]
```diff
@@ -269,7 +269,9 @@ def testInsertFieldValidatesMultipleDescriptors(self):
     self.assertEmpty(ns.GetFindings())
 
   def testAggregationDescriptorFailsWithoutAggregation(self):
-    """Check that aggregation descriptors fail without associated aggregation."""
+    """ Check that aggregation descriptors fail without associated
```
[the remark this answers is not in the document]
Comment (trav3711) [reply tools/validators/ontology_validator/yamlformat/tests/field_lib_test.py:272]: Done.

File: tools/validators/ontology_validator/yamlformat/validator/unit_lib.py (hunk 3 above)
Comment (trav3711) [reply tools/validators/ontology_validator/yamlformat/validator/unit_lib.py:235]: I think the docstring is in the right place. It's below def and above super().

File: tools/validators/ontology_validator/yamlformat/validator/subfield_lib.py (hunk 2 above)
Comment (trav3711) [reply tools/validators/ontology_validator/yamlformat/validator/subfield_lib.py:257]: The docstring is there for state_lib.py:154

**Review by charbull — CHANGES_REQUESTED** [review]
Can you please update: Attributes on the class, args on the init.

File: tools/validators/ontology_validator/yamlformat/validator/findings_lib.py [hunk 5]
```diff
@@ -332,70 +359,51 @@ class InconsistentFileLocationError(ValidationError):
...
-
-  def __init__(self, bad_text, context):
-    super(CapitalizationError,
-          self).__init__('"{0}" is capitalized incorrectly'.format(bad_text),
-                         context)
-
-
-class UnrecognizedFormatError(ValidationError):
-  """Validator does not know how to read content."""
+  Args:
```
[the remark this answers is not in the document]
Comment (trav3711) [reply tools/validators/ontology_validator/yamlformat/validator/findings_lib.py:390]: Done.

**Pull request closed without merging**